### ISSUE TITLE
'm' should mean "months" and not "minutes"

### ISSUE THREAD

**Issue by nanounanue** [dssg/triage#392]
In `interval` parameter in  the config file

**Comment by ecsalomon** [comment]
in postgres, `1m` is 1 minute. We used to convert to one month and throw a warning but decided to respect the behavior of postgres

**Comment by ecsalomon** [comment]
see https://github.com/dssg/timechop/pull/48

**Comment by nanounanue** [comment]
This is not true in the current version of triage. This is useful because it shortens the features names
